# Importing pandas to help in reading the uploaded file.

In [6]:
# import pandas
import pandas as pd 

In [7]:
aviation_data =pd.read_csv('aviation-accident-data-2023-05-16.csv')
aviation_data

date                       type registration  \
0        date unk.             Antonov An-12B       T-1206   
1        date unk.             Antonov An-12B       T-1204   
2        date unk.             Antonov An-12B       T-1201   
3        date unk.            Antonov An-12BK          NaN   
4        date unk.            Antonov An-12BP   CCCP-11815   
...            ...                        ...          ...   
23962  11-MAY-2023               Hawker 900XP       PK-LRU   
23963  11-MAY-2023  Cessna 208B Grand Caravan       PK-NGA   
23964  12-MAY-2023  Cessna 208B Grand Caravan       5X-RBR   
23965  14-MAY-2023            Boeing 747-4R7F       LX-OCV   
23966  15-MAY-2023                Learjet 35A       D-CGFQ   

                     operator fatalities             location  \
0               Indonesian AF        NaN                  NaN   
1               Indonesian AF        NaN                  NaN   
2               Indonesian AF        NaN                  NaN   
3                   Soviet AF        NaN  Tiksi Airport (IKS)   
4                   Soviet AF          0  Massawa Airport ...   
...                       ...        ...                  ...   
23962  Angkasa Super Services          0  Maleo Airport (MOH)   
23963  Nasional Global Aviasi          0    Fentheik Airstrip   
23964            Bar Aviation          0  Kampala-Kajjansi...   
23965                Cargolux          0  Luxembourg-Finde...   
23966                     GFD          2        Hohn Air Base   

               country cat     year  
0      Unknown country  U1  unknown  
1      Unknown country  U1  unknown  
2      Unknown country  U1  unknown  
3               Russia  A1  unknown  
4              Eritrea  A1  unknown  
...                ...  ..      ...  
23962        Indonesia  A2     2023  
23963        Indonesia  A2     2023  
23964           Uganda  A1     2023  
23965       Luxembourg  A2     2023  
23966          Germany  A1     2023  

[23967 rows x 9 columns]

PROJECT INTRODUCTION
# Analysis of the Aviation accident dataset
I used the aviation accident dataset provided to help i identifying trends and patterns in aviation safety. The following business problems were looked into:
        1. How have accident occurrences and fatalities changed across decades
        2. Which geographic locations/operators reported the most incidences
        3. Are the modern incidences(accidents) that occurred post 1970 less fatal than the earlier years



# Overview of the Dataset
This dataset contains 23967 rows and 9 columns that entails the following data information
        Date: date of the accident
        Type: the model/type of the aircraft involved
        Registration: the registration number of the aircraft
        Operator: the Airline or military operating the aircraft
        Fatalities: Number of casualties
        country & Location: the geographical location
        Year: year of accident
        